makeflow/src/makeflow.c

```diff
@@ -1118,6 +1118,7 @@ static void show_help_run(const char *cmd)
 	printf(" %-30s Advertise the master information to a catalog server.\n", "-a,--advertise");
 	printf(" %-30s Specify path to Amazon credentials (for use with -T amazon)\n", "--amazon-credentials");
 	printf(" %-30s Specify amazon-ami (for use with -T amazon)\n", "--amazon-ami");
+	printf(" %-30s Specify path to Container Image for Amazon-Batch (for use with -T amazon)\n", "--amazon-batch-image");
```

Comment:
Do we also need an option for the config file generated by `makeflow_amazon_batch_setup`?